core/src/epicli/data/common/defaults/configuration/applications.yml

```diff
@@ -124,3 +124,29 @@ specification:
         # Specify content for pgpool.conf
         # TODO: To be implemented
         # Example: https://github.com/CrunchyData/crunchy-containers/blob/master/examples/kube/pgpool/configs/pgpool.conf
+
+  - name: pgbouncer
+    enabled: yes
+    #https://github.com/brainsam/pgbouncer/tree/release/1.12
+    image_path: brainsam/pgbouncer:1.12
+    namespace: pgbouncer
+    service:
+      name: pgbouncer
+      port: 5432
+    replicas: 2
+    pgbouncer:
+      env:
+        target_address: pgpool.pgpool.svc.cluster.local #pgpool service name
+        target_port: "5432"
+        listen_address: "*"
+        listen_port: "5432"
+        authentication_file: "/etc/pgbouncer/auth/users"
+        authentication_type: md5
+        max_client_connections: "500"
+        default_pool_size: "25"
+        reserve_pool_size: "25"
+        pool_mode: transaction
+      secrets:
+        admin_users: postgres #pgbouncer admin user(s), eg: user1, user2
+        db_user: '"username"' #postgres username
```

Comment:
I think it would be better to have quotation in the jinja template of pgbouncer-users-file secret. But I'm working on different approach (synchronizing users with Postgresql instance).

Reply:
added to TODO, else this form will have to be changed at all